frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -23,11 +24,13 @@ const PerspectiveNav: React.FC<{}> = () => {
   const [perspective] = useActivePerspective();
   const allItems = useExtensions<PluginNavSection | NavItem | Separator>(isNavSection, isNavItem);
   const [pinnedResources, setPinnedResources, pinnedResourcesLoaded] = usePinnedResources();
+  const [validPinnedResources, setValidPinnedResources] = React.useState<string[]>(pinnedResources);
```

Comment:
This will lead back to [Bugzilla bug 2069685](https://bugzilla.redhat.com/show_bug.cgi?id=2069685). The current list of pinned resources may not be valid.

Reply:
```suggestion
  const [validPinnedResources, setValidPinnedResources] = React.useState<string[]>([]);
```